src/components/Editor/ItemEditor.vue

```diff
@@ -204,17 +221,30 @@ export default {
       var millisecond = parseInt(timeInISO.millisecond) || 0;
       return hour * 3600 + minute * 60 + second + millisecond / 1000;
     },
-    updateCorrectOption(option) {
+    updateCorrectOption(optionNumber) {
       // when some option is selected as correct, update it in the
       // item list
-      var indexOfSelectedOption = this.options.indexOf(option);
       this.localItemList[
         this.localSelectedItemIndex
-      ].details.correct_answer = indexOfSelectedOption;
+      ].details.correct_answer = optionNumber;
     },
+    deleteSelectedItem() {
+      this.$emit("delete-selected-item")
+    }
   },
 
   computed: {
+    deleteItemButtonTooltip() {
+      // tooltip text for delete item button
+      if (this.isPublished)
```

Comment:
comment everything and return empty string for now (to handle delete button bug)